runtime/src/lib.rs

```diff
@@ -96,6 +97,33 @@ pub fn native_version() -> NativeVersion {
 	}
 }
 
+/// Avoid processing transactions that are anything except staking and claims.
+///
+/// NOTE: This is only relevant for the initial PoA consensus-run-in period and should be removed
+/// for the normal chain runtime.
+#[derive(Default, Encode, Decode, Clone, Eq, PartialEq)]
+#[cfg_attr(feature = "std", derive(Debug))]
+pub struct OnlyStakingAndClaims;
+impl SignedExtension for OnlyStakingAndClaims {
+	type AccountId = AccountId;
+	type Call = Call;
+	type AdditionalSigned = ();
+	type Pre = ();
+	fn additional_signed(&self) -> rstd::result::Result<(), &'static str> { Ok(()) }
+	fn validate(
+		&self,
+		_who: &Self::AccountId,
+		call: &Self::Call,
+		_info: DispatchInfo,
+		_len: usize,
+	) -> Result<ValidTransaction, DispatchError> {
+		match call {
+			Call::Staking(_) | Call::Claims(_) => Ok(Default::default()),
+			_ => Err(DispatchError::NoPermission),
+		}
```

Comment:
CC especially here ^^ @jacogr 

(and acknowledging how cool this signed extension thingy is)

Question: why don't we add it to the beginning of the list then? they should execute in order afaik and if a call is not staking/claims we shouldn't check for nonce/fee etc.